webpack/assets/javascripts/react_app/components/common/CardTemplate/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CardItem from './CardItem';
+
+import { addGlobalFill } from '../../common/Fill/GlobalFill';
+
+const CardTemplate = ({ content, header, slotID, cardID }) => {
```

Comment:
This is still not used, right? Is it just bad rebase?

Reply:
bad rebase...